src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,85 @@
+<div th:fragment="navbar" id="navbar">
+    <nav class="navbar">
```

Comment:
You could add the role attribute for accessibility

Reply:
done